### ISSUE TITLE
Manually initialize

### ISSUE THREAD

**Issue by devnutmike** [ellisonleao/sharer.js#31]
How to manually init this so that it works inside dynamic content like Angular views?

**Comment by ellisonleao** [comment]
Sharer.js does not have an api , neither AMD, CommonJS support for now. How are you trying to use it? can you give some some example ?

**Comment by devnutmike** [comment]
On a full page load, DOM elements with the sharer class get assigned the click event. But for SPA type applications where views are loaded dynamically, changing to a view that has DOM elements with the sharer class, the click event does not get assigned because the code that does that doesn't run again.

I would like to use sharer buttons inside views and not just on the main html container page. 

Basically after loading a view, we would need a function exposed that will force another update to the DOM in order to assign the click events.

Share libraries like addThis account for this by exposing an init function or something similar.

**Comment by ellisonleao** [comment]
Yeah, that could be solved if i expose the `Sharer` class or maybe some of this methods. I will think about on something and let you know. The only thing i am not into right now is to add CommonJS / AMD support.

**Comment by devnutmike** [comment]
Okay great thanks. Really like this lightweight approach to share links and
being able to use your own buttons or links.

**Comment by ellisonleao** [comment]
hey @devnutmike , we got for now a npm package which probably handle what you need https://github.com/okmttdhr/sharer.npm.js

**Comment by ellisonleao** [comment]
Closing for now.
